src/traits/interface.rs

```diff
@@ -8,17 +8,21 @@ pub unsafe trait Interface: Sized + Abi {
     type Vtable;
     const IID: Guid;
 
+    /// Returns the vtable for the current interface.
     unsafe fn vtable(&self) -> &Self::Vtable {
         self.assume_vtable::<Self>()
     }
 
-    // TODO: name from assume_init
+    /// Returns the vtable for an assumed interface. The name comes from `Box`'s `assume_init` method as
```

Comment:
You might consider putting everything after the first sentence on a new line of the docs (even with an empty line in between). This tends to render much more nicely in rust-doc.

Reply:
https://github.com/microsoft/winrt-rs/issues/359